src/controllers/friend.controller.ts

```diff
@@ -29,6 +30,12 @@ export class FriendController {
     protected userRepository: UserRepository,
   ) {}
 
+  @ratelimit(true, {
```

Comment:
as we discuss before, we need to implement `[keyGenerator](https://github.com/nfriedly/express-rate-limit#keygenerator)` we use userid instead of IP address